src/Open3D/GUI/Window.cpp

```diff
@@ -544,8 +676,21 @@ void Window::OnMouseEvent(const MouseEvent &e) {
             break;
         case MouseEvent::WHEEL: {
             ImGuiIO& io = ImGui::GetIO();
-            io.MouseWheelH += (e.wheel.dx > 0 ? 1 : -1);
-            io.MouseWheel  += (e.wheel.dy > 0 ? 1 : -1);
+            float dx = 0.0, dy = 0.0;
+            if (e.wheel.dx != 0) {
+                dx = e.wheel.dx / std::abs(e.wheel.dx);
+            }
+            if (e.wheel.dy != 0) {
+                dy = e.wheel.dy / std::abs(e.wheel.dy);
+            }
+            // Note: ImGUI's documentation says that 1 unit is about 5 lines
+            if (e.wheel.isTrackpad) {
+                io.MouseWheelH += dx * 0.25;
```

Comment:
Maybe add constants to `e.wheel` for scaling input? Because there is no comment about scaling when trackpad is used

Reply:
Could you clarify what you mean about "no comment about scaling when trackpad is used"? The `if` says `isTrackpad` so it would seem to me like the scaling is documented. I don't think the scale factor should go in e.wheel, either, because the caller (the run loop in Application) doesn't know about how Window needs to interpret the event. If I remember correctly, moving an actual mouse wheel produces a dy of 1, and moving the trackpad one "trackpad unit" produces a dy of 1. But if you are scrolling via trackpad you want to move less than if you are wheeling on a mouse. However, we don't know what scale factor to use until we know that we are passing it to ImGUI. Not sure if that answers your question or not?

Reply:
It's all clear with comments about units and lines, thanks.
What I'm trying to say is regarding to `0.25` constant, I think it's better to make it public. 
> But if you are scrolling via trackpad you want to move less than if you are wheeling on a mouse

This, also makes me thinking that public constant will be good. Why not allow user to control degree of scaling input from touchpad.